crazyflie/launch/launch.py

```diff
@@ -47,16 +39,19 @@ def generate_launch_description():
     with open(motion_capture_yaml, 'r') as ymlfile:
         motion_capture = yaml.safe_load(ymlfile)
 
-    # motion_capture_params = motion_capture["/motion_capture_tracking"]["ros__parameters"]
```

Comment:
Why are you adding in motion capture here

Reply:
This was in the crazyflie branch too